docker/compute_worker/compute_worker.py

```diff
@@ -371,6 +374,21 @@ def _get_bundle(self, url, destination):
 
             # Input from submission (or submission + ingestion combo)
             docker_cmd += ['-v', f'{self.input_dir}:/app/input']
+            if self.detailed_results_url and self.is_scoring and kind == 'program':
+                # we have a detailed results url, we are in the scoring step, and we aren't the ingestion program
+                detail_path = os.path.join(self.output_dir, "detailed_results.html")
+
+                # Is there a better way to create the entire path without having to mkdir each incremental step?
+                if not os.path.exists(detail_path):
+                    if not os.path.exists(self.output_dir):
+                        os.mkdir(self.output_dir)
+                    open(detail_path, 'a').close()
```

Comment:
this makes the file so you can chmod it, I assume? could maybe make this more clear like:

```python
# Create detailed file and set perms so we can read the file to send to the sas URL
open(detail_path, 'a').close()
os.chmod(detail_path, 0o777)
```